import-automation/import-progress-dashboard-frontend/client/.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('gts/.prettierrc.json')
```

Comment:
where is the gts folder?

Reply:
Removed. I could not configure gts correctly and switched to eslint.